core/server/server.go

```diff
@@ -64,47 +74,27 @@ func (s *Server) PostTx(ctx context.Context, in *pb.TxStatus) (*pb.CommonReply,
 			s.mg.P2pSvr.SendMessage(context.Background(), msg, opts...)
 		}()
 	}
+	matricLabelsOut := prom.Labels{
+		"bcname": in.GetBcname(),
+		"type":   "PostTx",
+	}
+	matrics.DefaultServerMetrics.RPCFlowOut.With(matricLabelsOut).Add(float64(proto.Size(out)))
```

Comment:
Can grcp interceptors be used to reduce duplicate code? Examples: UnaryServerInterceptor.

Reply:
+1